cogs/mod.py

```diff
@@ -152,7 +152,7 @@ def check_if_target_is_staff(self, target):
     @commands.guild_only()
     @commands.bot_has_permissions(ban_members=True)
     @commands.check(check_if_staff)
-    @commands.command(aliases=["yeet"])
+    @commands.command(aliases=["yeet","softban","silentban"])
```

Comment:
Please put spaces between `,` and `"`, as per PEP8.

Similarly, I don't think that we can merge hackban into ban at this point (for the reasons I mentioned on #63), but I do agree that it's a good idea to move silentban into ban.

Reply:
My bad, I'll get that fixed.

Reply:
My bad, I'll get that fixed.